.gitignore

```diff
@@ -16,4 +17,5 @@ npm-debug.log
 gen/
 .nyc_output/
 coverage/
-package-lock.json
+.rpt2_cache
+dist
```

Comment:
I'm not sure if the `dist` folder should be removed!? 
I guess the `dist` directory only would be useful if someone whants to use the DataView class as a standalone and load it e.g. from cdnjs. Whe should discuss this further.

Reply:
Oh, this is still unresolved.
Maybe it would be better to start using releases, committing dist seems messy.
Other option that already works is JSDELIVR which (among other things) uses npm releases:
```html
<script src="https://cdn.jsdelivr.net/npm/vis-data"></script>
<script src="https://cdn.jsdelivr.net/npm/vis-data@5.0.0"></script>
<script src="https://cdn.jsdelivr.net/npm/vis-data@6"></script>
```

Reply:
I'm ok with the `jsdelivr` for now. We will see what peaople are going to expect.